mint-nft/index.js

```diff
@@ -16,6 +17,11 @@ const __filename = fileURLToPath(import.meta.url);
 const __dirname = path.dirname(__filename);
 /* eslint-enable no-underscore-dangle */
 
+const LIKER_NFT_FEE_WALLET = 'like10ywsmztkxjl55xarxnhlxwc83z9v2hkxtsajwl';
+const royaltyRateBasisPoints = 1000; // 10% as in current chain config
+const royaltyFeeAmount = 25000; // 2.5%
+const royaltyTotalAmount = 975000; // 1000000 - fee
```

Comment:
Why not use `1000000 - royaltyFeeAmount` directly?
Or more generally:
```JS
const royaltyFullWeight = 1000000;
const royaltyFeeAmount = 0.025 * royaltyFullWeight;
const royaltyTotalAmount = royaltyFullWeight - royaltyFeeAmount;
```

Also the `total` amount is not actually `total`, I suggest something like `royaltyUserAmount`